backend/src/services/AttendanceService.ts

```diff
@@ -123,15 +123,28 @@ export class AttendanceService {
     return points;
   }
 
-  async saveAttendance(attendance: Attendance): Promise<Attendance | undefined> {
+  async saveAttendance(
+    attendanceQuery: GetAttendanceQuery,
+    attendanceReq: AttendanceRequest,
+    officerId: number
+  ): Promise<Attendance> {
     const attendanceRepository = getRepository(Attendance);
```

Comment:
This might just be a 'me' thing, but maybe a reminder of what getRepository does and why we pass in Attendance?

Reply:
I think this is more appropriate as an information to be added in a documentation, which is already done [here](https://github.com/HKN-UCSD/hkn-server/blob/master/backend/guides/add/add_new_service.md). We use getRepository() in an ad hoc manner in a lot of places so it's kinda hard to put a comment for all of them